packages/braid-design-system/src/lib/components/useToast/Toaster.tsx

```diff
@@ -22,25 +27,55 @@ export const Toaster = ({ toasts, removeToast }: ToasterProps) => {
     [remove, removeToast],
   );
 
+  useEffect(() => {
+    if (toasts.length <= 1) {
+      setExpanded(false);
+    }
+  }, [toasts.length]);
+
   return (
-    <Box
-      position="fixed"
-      zIndex="notification"
-      width="full"
-      pointerEvents="none"
-      paddingX="gutter"
-      bottom={0}
-    >
-      {toasts.map(({ toastKey, ...rest }) => (
-        <Box key={toastKey} paddingBottom="small">
-          <ToastComponent
-            ref={itemRef(toastKey)}
-            toastKey={toastKey}
-            onClose={onClose}
-            {...rest}
-          />
+    <>
+      {isMobileDevice && expanded && toasts.length > 1 && (
+        <Box
+          position="fixed"
+          inset={0}
+          zIndex="notification"
+          onClick={() => setExpanded(false)}
+        />
+      )}
+
+      <Box
+        display="flex"
+        justifyContent="center"
+        position="fixed"
+        bottom={0}
+        width="full"
+        zIndex="notification"
+      >
+        <Box
+          width="full"
+          maxWidth={toastWidth}
+          pointerEvents={toasts.length === 0 ? 'none' : undefined}
+          marginX="gutter"
+          paddingBottom="xsmall"
```

Comment:
I believe it is these properties resulting in the screenshot diff (even though they dont result in a visual).

The `pointerEvents` change looks like it was working around this permanent space too. Happy to pair on this